src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
```

Comment:
Here are the descriptions of each from the ICU header files:
* ChineseCalendar is a concrete subclass of Calendar that implements a traditional Chinese calendar.  The traditional Chinese calendar is a lunisolar calendar
* JapaneseCalendar is a subclass of GregorianCalendar that numbers years and eras based on the reigns of the Japanese emperors.  The Japanese calendar is identical to the Gregorian calendar in all respects except for the year and era.